source/documentation/services/SSL-certificate-management.html.md.erb

```diff
@@ -0,0 +1,46 @@
+---
+title: Gandi.net SSL Certificates
+last_reviewed_on: 2021-01-04
+review_in: 3 months
+---
+
+# Gandi.net SSL Certificates
+
+This processes is for any Ministry of Justice users (or suppliers) requiring a Gandi.net SSL Certificate.
+
+There are 2 categories of certificate we use at Ministry of Justice:
+
+* AWS Certificate Manager (ACM)/Let’s Encrypt - automated certificate management for modern cloud native software and infrastructure
+* Gandi.net - where automated certificate management is not possible
```

Comment:
Is it worth mentioning DigiCert? 🙈

Reply:
I had intentionally left it out as it's no longer an option for GSI domains. I felt that if I included it as an option then users my think these are still available.

Reply:
Yeah, that makes sense 👍